0300-method.Rmd

```diff
@@ -152,24 +152,24 @@ Fish passage potential was determined for each stream crossing identified as a c
 <br>
 
 ```{r tab-barrier-scoring, eval=F}
-tab <- as_tibble(t(tab_barrier_scoring)) %>% 
-  mutate(V4 = names(tab_barrier_scoring)) %>% 
+tab <- as_tibble(t(fpr_table_barrier_scoring)) %>% 
```

Comment:
Why not call the tables from the fpr functions (ex fpr::fpr_table_habvalue and fpr::fpr_table_barrier_scoring )?